library/Auth.re

```diff
@@ -0,0 +1,31 @@
+exception Token_secret_not_provided;
+
+let getTokenSecret = () =>
+  try (Sys.getenv("JWT_SECRET")) {
+  | Not_found => raise(Token_secret_not_provided)
+  };
+
+let decodeToken = token => {
+  open Jwt;
+  let payload = t_of_token(token) |> payload_of_t;
+  find_claim(sub, payload);
+};
+
+let encodeToken = userId => {
+  open Jwt;
+  let header = header_of_algorithm_and_typ(HS256(getTokenSecret()), "JWT");
+  let payload =
+    empty_payload
+    |> add_claim(iss, "summhub_server")
```

Comment:
let's remove this iss claim next to simplify

Reply:
okay